api-module-library/hubspot/api.js

```diff
@@ -688,6 +688,28 @@ class Api extends OAuth2Requester {
         return this._patch(options);
     }
 
+    async pushLandingPageDraftToLive(objId){
+        const options = {
+            url: `${this.baseUrl}${this.URLs.landingPageById(objId)}/draft/push-live`,
+        };
+        return this._post(options);
+    }
+
+    async publishLandingPage(objId, publishDate) {
+        const options = {
+            url: `${this.baseUrl}${this.URLs.landingPages}/schedule`,
+            body: {
+                id: objId,
+                publishDate
+            },
+            headers: {
+                'Content-Type': 'application/json',
+                Accept: 'application/json',
+
+            }
```

Comment:
The suggested approach above would be backward compatible, I think.

Reply:
We could override _post to apply these headers, or look to change the Requester class to default to add these headers. But, given that some _post requests, even just in this api module, do not take these headers, I like the explicit addition rather than needing to explicitly subtract them. I imagine sending them regardless would not cause issue most of the time.

Maybe it would make sense to add a new Requester method like _postJson() which added these headers?

Reply:
@MichaelRyanWebber - let's keep this as is for now and try to get back to it later.

Reply:
@hudsontavares  Sounds good, added
https://linear.app/lefthook/issue/LEF-442/re-assess-requests-handling-of-headers-on-postpatchput